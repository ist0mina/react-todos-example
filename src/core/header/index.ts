export * from './Header';
export * from './HeaderContainer';
export * from './types';