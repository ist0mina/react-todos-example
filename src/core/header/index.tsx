import React from 'react';
import { Jumbotron } from 'react-bootstrap';

const Header = () => {
    return (
        <Jumbotron>
            <h1 className="text-center">Todo list</h1>
        </Jumbotron>
    );
};

export default Header;