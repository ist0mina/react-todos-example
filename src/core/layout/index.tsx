import React from 'react';
import Header from '../header';

interface LayoutProps {
    children: React.ReactNode;
};

const Layout: React.FC<LayoutProps> = ({ children }) => (
    <div>                
        <Header />
        { children}
    </div>
);

export default Layout;