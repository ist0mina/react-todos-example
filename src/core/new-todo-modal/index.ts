export * from './NewTodoModal';
export * from './NewTodoModalContainer';
export * from './types';