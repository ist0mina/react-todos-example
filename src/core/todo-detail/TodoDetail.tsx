import React from 'react';

export const TodoDetail: React.FC = () => {
    return (
        <div>Todo detail</div>
    );
}