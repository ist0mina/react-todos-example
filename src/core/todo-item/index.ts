import TodoItemContainer from './TodoItemContainer';

export * from './types';

export default TodoItemContainer;
