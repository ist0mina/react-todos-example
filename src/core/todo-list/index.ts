import TodoListContainer from './TodoListContainer';

export * from './types';

export default TodoListContainer;