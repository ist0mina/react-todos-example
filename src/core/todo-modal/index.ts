import TodoModalContainer from './TodoModalContainer';

export * from './types';
export default TodoModalContainer;