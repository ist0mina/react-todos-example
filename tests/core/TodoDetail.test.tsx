import { TodoDetail } from '../../src/core/todo-detail';

import { reduxify, makeMountrender, snapshotify } from '../utils';

describe('<TodoDetail>', () => {
    it('matches snapshots', () => {
        const component = reduxify({ Component: TodoDetail });
        const wrapper = makeMountrender(component)();

        expect(snapshotify(wrapper)).toMatchSnapshot();
    });
});