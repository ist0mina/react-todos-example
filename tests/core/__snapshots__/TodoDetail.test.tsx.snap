// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TodoDetail> matches snapshots 1`] = `"<div>Todo detail</div>"`;
